integration_tests/01_storage_retrieval_ok.sh

```diff
@@ -14,8 +14,8 @@ MINER=t01000
 dealbot --api $TOKEN:$API storage-deal --data-dir $DATADIR --miner $MINER
```

Comment:
Alternatively we could just remove the 2nd storage deal - no need to wait for 2 storage deals on CI, it will just make the suite slower.